requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/hildensia/bayesian_changepoint_detection@2dd95f5c1d028116899a842ccb3baa173f9d5be9#egg=bayesian-changepoint-detection
+bayesian-changepoint-detection @ git+https://github.com/hildensia/bayesian_changepoint_detection@2dd95f5c1d028116899a842ccb3baa173f9d5be9#egg=bayesian-changepoint-detection
```

Comment:
B: My assumption was we are also going to remove it from the requirements.txt file since that gets used inside setup.py file.

Reply:
The build is failing. I added this back to test if it is due to this